frx_challenges/web/views/versions.py

```diff
@@ -77,6 +77,7 @@ def download_results(request: HttpRequest, id: int) -> HttpResponse:
 def view(request: HttpRequest, id: int) -> HttpResponse:
     version = Version.objects.get(id=id)
     evaluation = version.latest_evaluation
+    is_collaborator = _validate_collaborator(request, version.submission.id)
```

Comment:
Should this be `is_collaborator = _validate_collaborator(request, id)` like line 58?

Reply:
```suggestion
    is_collaborator = _validate_collaborator(request, id)
```